documentation/documentation.html

```diff
@@ -15,16 +15,16 @@
     <title>FeedyUI - Component Library</title>
   </head>
   <body>
-    <iframe
-      id="avatar-iframe"
-      src="./avatar.html"
+    <!-- <iframe
+      id="alert-iframe"
+      src="./alert.html"
```

Comment:
Remove commented code

Reply:
Documentation site code. Commented to help review check my changes upfront instead of scrolling.